cime/config/e3sm/config_archive.xml

```diff
@@ -69,63 +85,65 @@
       <rpointer_file>rpointer.ice$NINST_STRING</rpointer_file>
       <rpointer_content>./$CASE.cice$NINST_STRING.r.$DATENAME.nc</rpointer_content>
     </rpointer>
+    <test_file_names>
+      <tfile disposition="copy">rpointer.ice</tfile>
+      <tfile disposition="copy">casename.cice.r.1976-01-01-00000.nc</tfile>
+      <tfile disposition="move">casename.cice.h.1976-01-01-00000.nc</tfile>
+    </test_file_names>
   </comp_archive_spec>
 
   <comp_archive_spec compname="mpascice" compclass="ice">
     <rest_file_extension>rst</rest_file_extension>
+    <rest_file_extension>rst.am.timeSeriesStatsMonthly</rest_file_extension>
     <hist_file_extension>hist</hist_file_extension>
     <rest_history_varname>unset</rest_history_varname>
     <rpointer>
       <rpointer_file>rpointer.ice$NINST_STRING</rpointer_file>
       <rpointer_content>$MPAS_DATENAME</rpointer_content>
     </rpointer>
-  </comp_archive_spec>
-
-  <comp_archive_spec compname="dice" compclass="ice">
```

Comment:
@mfdeakin-sandia , a lot of data model archive configurations were removed. Was this intentional?

Reply:
Yes, sorry, I wasn't in on Friday so didn't get the email for this. This was needed since E3SM and CESM share the same data models, so their archive entries were causing xml conflicts with the new tests.